config/initializers/booking_bug_config.rb

```diff
@@ -0,0 +1,36 @@
+module Config
+  def self.included(base)
+    base.const_set(:Config, Class.new(::Config::Config))
+    base::Config.attributes = base::ATTRIBUTES
+    base.cattr_accessor :config
+  end
+
+  class Config
+    cattr_accessor :attrs
+
+    def self.attributes=(attributes)
+      self.attrs = attributes
+      attr_accessor(*attributes)
+    end
+
+    def initialize
+      yield(self) if block_given?
+    end
+
+    def attributes
+      self.class.attrs
+    end
+  end
+end
+
+BookingBug::ATTRIBUTES = %w(environment company_id api_key app_id email password).freeze
```

Comment:
Why are we telling `BookingBug` how it should define it's configuration? Isn't it `BookingBug`'s responsibility to define it's interface for configuration?